services/ocr/utils/blackListChecker.js

```diff
@@ -0,0 +1,12 @@
+import { getMysqlClient } from '../../../utils/index.js';
+import { selectBlackList } from '../../../utils/mysql-queries/index.js';
+
+// return true if text doesn't contain blocked phrases
+export const blackListChecker = async (text) => {
+    const mysql = await getMysqlClient();
+    const blackList = await selectBlackList(mysql);
+    if (!blackList) return true;
```

Comment:
make selectBlackList throw an exception if no list is present and remove this check

Reply:
Hm. This is an easy way to tell developer: you should setup a blacklist. Good idea.